scalyr_agent/configuration.py

```diff
@@ -259,6 +259,27 @@ def parse_monitor_config(self, monitor_config, context_description='uncategorize
 
         return monitor_config
 
+    # k8s cache options
+    @property
+    def k8s_ignore_namespaces(self):
+        return self.__get_config().get_string('k8s_ignore_namespaces')
+
+    @property
+    def k8s_api_url(self):
+        return self.__get_config().get_string('k8s_api_url')
+
+    @property
+    def k8s_verify_api_queries(self):
+        return self.__get_config().get_bool('verify_k8s_api_queries')
```

Comment:
Done.  Only reason I didn't do this earlier was because it was an existing parameter and I wasn't sure if changing it would break anyone's current setup.